axelrod/strategies/_strategies.py

```diff
@@ -72,6 +72,7 @@
 from .selfsteem import SelfSteem
 from .shortmem import ShortMem
 from .stalker import Stalker
+from .axelrod_first import SteinAndRapoport
```

Comment:
Lines 7,8,9 of this file look like:

```
  7 from .axelrod_first import (                                                                                                                      
  8     Davis, RevisedDowning, Feld, Grofman, Nydegger, Joss, Shubik, Tullock,                                                                        
  9     UnnamedStrategy)  
```

So you can remove this line and just add `SteinAndRapoport` to the list imported there.